### ISSUE TITLE
Installation per se Readme fails on Windows

### ISSUE THREAD

**Issue by tkuenzmw** [mcdougallab/matlabneuroninterface#107]
The instructions to "1. make sure Neuron is installed" at least on Windows tempt the user to download the current stable Neuron 8.2.
However, I cannot get the MATLAB Neuron Toolbox to work with this. Instead, a specific nightly build of Neuron 9 seems to be required. A public user has no way of getting this currently,right?

**Comment by ramcdougal** [comment]
This is correct. We have started a project to make it work with the regular 9, but we're not there yet. (At an earlier point it did work with 8.x, but it no longer works with that either.)

I'll add a link to the special version needed so someone can at least use it for now with Windows.

We also need to add text explaining that mingw needs to be installed in Windows. I know the 6.3 version available through MATLAB works.

**Comment by tkuenzmw** [comment]
@ramcdougal temporarily updating the Readme (and linking the working nightly installer) would be a good interim solution, Robert. By the way in MATLAV R2023b and later mingw 8.1 is the default. I can build the interface successfully with that in 24a, so that should be ok.

**Comment by 33729468** [comment]
Good day,
I am a Master's student in Bio-engineering at the University of Pretoria. I intend to use NEURON within MATLAB to model neural networks of the cochlear nucleus that receive input from a peripheral model also implemented in MATLAB. However, I am encountering significant difficulties in implementing the NEURON toolbox for MATLAB. Could you advise on any version incompatibilities that I should be aware of? Specifically, are there particular versions of MATLAB, NEURON, and MinGW that are recommended to ensure compatibility when using NEURON within MATLAB?

**Comment by thkupy** [comment]
> Good day, I am a Master's student in Bio-engineering at the University of Pretoria. I intend to use NEURON within MATLAB to model neural networks of the cochlear nucleus that receive input from a peripheral model also implemented in MATLAB. However, I am encountering significant difficulties in implementing the NEURON toolbox for MATLAB. Could you advise on any version incompatibilities that I should be aware of? Specifically, are there particular versions of MATLAB, NEURON, and MinGW that are recommended to ensure compatibility when using NEURON within MATLAB?

Hi @33729468! That sounds like an interesting project - I have been working on modeling neurons in the CN for a good part of my scientific career as well. If you want you can drop me an email at tkuenzel[at]mathworks.com and we can talk about the project a bit more.

Regarding the installation: if you are on windows there are currently some "specifics", especially regarding the version of NEURON you need to install in order to be able to build the Neuron-MATLAB Interface. @ramcdougal can provide the link to a working installer of a specific build - at some point it would be good to add this to the README as a interim solution

**Comment by tkuenzmw** [comment]
> Good day, I am a Master's student in Bio-engineering at the University of Pretoria. I intend to use NEURON within MATLAB to model neural networks of the cochlear nucleus that receive input from a peripheral model also implemented in MATLAB. However, I am encountering significant difficulties in implementing the NEURON toolbox for MATLAB. Could you advise on any version incompatibilities that I should be aware of? Specifically, are there particular versions of MATLAB, NEURON, and MinGW that are recommended to ensure compatibility when using NEURON within MATLAB?

Hi @33729468! That sounds like an interesting project - I have been working on modeling neurons in the CN for a good part of my scientific career as well. If you want you can drop me an email at tkuenzel[at]mathworks.com and we can talk about the project a bit more.

Regarding the installation: if you are on windows there are currently some "specifics", especially regarding the version of NEURON you need to install in order to be able to build the Neuron-MATLAB Interface. @ramcdougal can provide the link to a working installer of a specific build - at some point it would be good to add this to the README as a interim solution


EDIT: reposting from correct account
